pkg/console/controllers/route/controller.go

```diff
@@ -224,6 +233,24 @@ func (c *RouteSyncController) SyncCustomRoute(operatorConfig *operatorsv1.Consol
 	return customRoute, "", customRouteError
 }
 
+func (c *RouteSyncController) ValidateDefaultRouteConfig(operatorConfig *operatorsv1.Console) (*corev1.Secret, error) {
```

Comment:
Maybe `GetDefaultRouteTLSConfig`. This isn't really validating anything just returning config, specifically TLS config.